.travis.yml

```diff
@@ -65,6 +65,8 @@ before_script:
     - mysql LorisTest < SQL/0000-00-03-ConfigTables.sql
     - mysql LorisTest < SQL/0000-00-04-Help.sql
     - mysql LorisTest < docs/instruments/radiology_review.sql
+    - mysql LorisTest < SQL/Release_patches/15.10_To_16.0_upgrade.sql
+    - mysql LorisTest < SQL/Release_patches/16.0_To_16.1_upgrade.sql
```

Comment:
These shouldn't be required because the schema file should already have the changes all incorporated.

What changes are missing?